PublicMapping/districtbuilder
# Update map color list

## Overview

Replace the color list that dictates the colors of districts. I hand selected visually distinct
and appealing colors for the first 30. I increased the total number of colors to 400+, which will
handle all US state house contests (NH has 400 reps), and varied the appearance of similar colors (for example, avoided having two districts in a row have varieties of greens.

### Demo

<img width="1209" alt="Screen Shot 2020-08-15 at 8 24 11 AM" src="https://user-images.githubusercontent.com/1809908/90396922-0fd29880-e065-11ea-8fdb-12f10335d040.png">

### Notes

Used this helpful post as a starting point: https://stackoverflow.com/questions/33295120/how-to-generate-gif-256-colors-palette/33295456#33295456

## Testing Instructions

- Open a mapping project
- Confirm that the new colors are appearing
